obal/data/playbooks/library/rpmspec_changelog_check.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+from ansible.module_utils.basic import AnsibleModule
+
+import os
+import subprocess
+import glob
+
+def run_module():
+    module_args = dict(
+        directory=dict(type='str', required=True),
+    )
+
+    result = dict(
+        changed=False,
+        changelog_version_release='',
+        specfile_version_release=''
+    )
+
+    module = AnsibleModule(
+        argument_spec=module_args,
+        supports_check_mode=True
+    )
+
+    if module.check_mode:
+        return result
+
+    specfile = glob.glob(os.path.join(module.params['directory'], "*.spec"))[0]
+
+    cmd = ['rpm', '--query', '--changelog', '--specfile', specfile]
+    result['changelog_version_release'] = subprocess.check_output(cmd).split("\n")[0].split(" ")[-1]
+
+    cmd = ['rpm', '--query', '--queryformat', '%{version}-%{release}\n', '--undefine', 'dist', '--specfile', specfile]
```

Comment:
If you leave out the `\n` at the end of the format, can you remove the split on the next line?

Reply:
True, but then i get `0.4.5-20.4.5-20.4.5-20.4.5-2` because I guess `rpm` parses every instance in the spec? I'm not sure why it gets output multiple times.

Reply:
If you add `--srpm` it will not output the version-release for both and give you a single entry.

Reply:
Indeed, otherwise it will print it for every subpackage (like `-doc`) as well.